baremetal/metal3machine_manager.go

```diff
@@ -53,11 +53,12 @@ const (
 	ProviderName = "baremetal"
 	// HostAnnotation is the key for an annotation that should go on a Machine to
 	// reference what BareMetalHost it corresponds to.
-	HostAnnotation     = "metal3.io/BareMetalHost"
-	requeueAfter       = time.Second * 30
-	bmRoleControlPlane = "control-plane"
-	bmRoleNode         = "node"
-	userDataFinalizer  = "metal3machine.infrastructure.cluster.x-k8s.io/userData"
+	HostAnnotation      = "metal3.io/BareMetalHost"
+	requeueAfter        = time.Second * 30
+	bmRoleControlPlane  = "control-plane"
+	bmRoleNode          = "node"
+	userDataFinalizer   = "metal3machine.infrastructure.cluster.x-k8s.io/userData"
+	pausedAnnotationKey = "metal3.io/capM3"
```

Comment:
nit, maybe capm3, to be consistent with the other places where we use it ?

Reply:
ok. I'll change it.